nominees/upyog.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "UPYOG",
+  "aliases": [
+    "Urban Platform for Delivery of online Governance"
+  ],
+  "description": "The Centre for Digital Governance at the National Institute of Urban Affairs has launched the Urban Platform for deliverY of Online Governance programme as one of the flagship initiatives under the National Urban Digital Mission. The UPYOG program aims at supporting states in implementing integrated e-governance and digital delivery of municipal services to citizens",
+  "website": "https://upyog.niua.org/",
+  "license": [
+    {
+      "spdx": "MIT",
+      "licenseURL": "https://github.com/upyog/UPYOG/blob/master/LICENSE"
+    }
+  ],
+  "SDGs": [
+    {
+      "SDGNumber": 11,
+      "evidenceText": "The objective of the program is to build and manage a \"shared digital good\" called UPYOG, comprising a set of software building blocks as well as reference applications for most needed e-governance solutions in the urban context."
+    }
+  ],
+  "sectors": [],
+  "type": [
+    "software"
+  ],
+  "repositories": [
+    {
+      "name": "main",
+      "url": "hhttps://github.com/upyog/UPYOG"
```

Comment:
As @conradsp mentioned this looks like a fork of an existing DPG -> [DIGITI](https://github.com/egovernments/DIGIT-OSS) are there significant changes to justify it as a different digital solution or is it just an implementation of DIGIT?